plugin/pkg/scheduler/factory/plugins.go

```diff
@@ -213,6 +218,14 @@ func RegisterAlgorithmProvider(name string, predicateKeys, priorityKeys util.Str
 		FitPredicateKeys:     predicateKeys,
 		PriorityFunctionKeys: priorityKeys,
 	}
+	temp_name := ""
+	if providerCounter == 1 {
+		temp_name = " | " + name + " | "
+	} else {
+		temp_name = name + " | "
+	}
+	providerCounter += 1
+	AlgorithmProviders += temp_name
```

Comment:
Instead of adding a flag, please just write a function that does this join (ideally using strings.Join) and call it when printing the description of the algorithm_provider flag.